### DISCUSSION TITLE
More Extra Effects for spells.

### DISCUSSION THREAD

**Discussion by Guite-Emortal** [AscensionGameDev/Intersect-Engine#1476] [Ideas]
**Is your feature request related to a problem? Please describe.**
Yes. It's a little frustrating that the Equipment Bonus Effects [Cooldown Reduction, Life Steal, Tenacity, Luck, and EXP] are not available in the spells editor. 

**Describe the solution you'd like**
Please copy those effects into the spell editor.

**Additional context**
I personally want to make Scrolls of Bonus EXP that give 50% Bonus EXP for 15/30/60min, but my own personal interests aside it would create more diverse buffs for all games.

**Comment by Spectreye** [comment]
I support this proposal ! i actually use "equipment buff" but it would be very cool to have these effect on spell too !

**Comment by Uyarrr** [comment]
100% Agree

**Comment by WeylonSantana** [comment]
I already did it, it's doable, I created a new nud that controls the effect percentage
In addition to the effects provided by the armor, it would be interesting to change the attack speed, or walking, confusion, effects for spells that are doable from the Alex fork.
and
Extra effect for spells and armors could be damage reduction, damage increase and damage reflection
